### PR TITLE
RSDK-8714: add Servo wrappers

### PR DESCRIPTION
**Pull request by gloriacai01**
Add servo component wrappers in java sdk + associated tests

### CODE REVIEW

File: core/sdk/src/main/java/com/viam/sdk/core/component/servo/ServoRPCService.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    @Override
+    public void getGeometries(com.viam.common.v1.Common.GetGeometriesRequest request,
+                     StreamObserver<com.viam.common.v1.Common.GetGeometriesResponse> responseObserver){
+        Servo servo = getResource(Servo.named(request.getName()));
+        List<Common.Geometry> geometries = servo.getGeometries(Optional.ofNullable(request.getExtra()));
+        responseObserver.onNext(Common.GetGeometriesResponse.newBuilder().addAllGeometries(geometries).build());
+        responseObserver.onCompleted();
+
+    }
```
Comment (njooma) [inline core/sdk/src/main/java/com/viam/sdk/core/component/servo/ServoRPCService.java:66]: This file is missing `doCommand`

File: core/sdk/src/test/kotlin/com/viam/sdk/core/component/servo/ServoRPCClientTest.kt [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    @Test
+    fun getPosition() {
+        `when`(servo.getPosition(any(Struct::class.java) ?: Struct.getDefaultInstance())).thenReturn(80)
+        val pos = client.getPosition()
+        verify(servo).getPosition(Struct.getDefaultInstance())
+        assertEquals(80, pos)
+    }
+
```
Comment (njooma) [inline core/sdk/src/test/kotlin/com/viam/sdk/core/component/servo/ServoRPCClientTest.kt:81]: Missing the test for `doCommand`

**Review by njooma — CHANGES_REQUESTED** [review]
Almost! Seems like you forgot `doCommand` in a few places. But overall this is great!

**Review by lia-viam — APPROVED** [review]

**Comment by lia-viam** [comment]
I think this needs a rebase but other than that you're probably good to merge!

**Review by purplenicole730 — APPROVED** [review]
For better recording purposes, please include the ticket number in the title of this PR. (It also helps with integrating with Jira)

ex: RSDK-XXXX: add servo wrappers

**Pull request merged**